source/deployment/customize-email.rst

```diff
@@ -181,10 +323,123 @@ SendInviteEmails
++===========+====================================================================+========================================+
+| SiteURL   | URL of the Mattermost server                                       | --                                     |
++-----------+--------------------------------------------------------------------+----------------------------------------+
+| Title     | Message heading                                                    | api.templates.invite_body.title        |
++-----------+--------------------------------------------------------------------+----------------------------------------+
+| Info1     | Message body                                                       | api.templates.invite_body.info         |
++-----------+--------------------------------------------------------------------+----------------------------------------+
+| Button    | Button for confirmation                                            | api.templates.invite_body.button       |
++-----------+--------------------------------------------------------------------+----------------------------------------+
+| ExtraInfo | Additional info about Mattermost                                   | api.templates.invite_body.extra_info   |
++-----------+--------------------------------------------------------------------+----------------------------------------+
+| TeamURL   | URL to the team the user has been invited to                       | --                                     |
++-----------+--------------------------------------------------------------------+----------------------------------------+
+| Link      | URL for team invite confirmation (not to be confused with TeamURL) | --                                     |
++-----------+--------------------------------------------------------------------+----------------------------------------+
+
+
+NotificationEmailBody
+~~~~~~~~~~~~~~~~~~~~~
+
+**Purpose**:
+Sent to the user as a notification for new messages or mentions.
+
+**Body Props**:
+
++----------+------------------------------+---------------------------------+
+| Prop     | Content                      | i18n String                     |
++==========+==============================+=================================+
+| SiteURL  | URL of the Mattermost server | --                              |
++----------+------------------------------+---------------------------------+
+| Button   | Button to post               | api.templates.post_body.button  |
++----------+------------------------------+---------------------------------+
+| TeamLink | URL to Team                  | --                              |
++----------+------------------------------+---------------------------------+
+
+
+This email can change depending on settings and type of channel the notification is sent for.
+
+**For group channels**:
+**With full notification contents enabled**: 
```

Comment:
In preview these two headings are in one line - might need a space in between them.

(**For group channels**:
**With full notification contents enabled**: )